include/openthread/commissioner.h

```diff
@@ -110,6 +110,18 @@ typedef struct otCommissioningDataset
     bool mIsJoinerUdpPortSet : 1; ///< TRUE if Joiner UDP Port is set, FALSE otherwise.
 } otCommissioningDataset;
 
+/**
+ * This structure represents a Joiner Info.
+ *
+ */
+typedef struct otJoinerInfo
+{
+    otExtAddress mEui64;                 /// Joiner eui64
+    char         mPsk[OT_PSKC_MAX_SIZE]; /// Joiner pskc
```

Comment:
I think this should be PSKd  and won't necessaily be the same size as PSKc
`OT_PSKC_MAX_SIZE` is 16 bytes, but PSKd can be up `kPskMaxLength = 32`.